_videos/2018-02-23-software-as-a-craft.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Raquel M Carmena
+layout: video
+name: software-as-a-craft
+image:
+ src: /assets/custom/img/videos/software-as-a-craft.png
+title: "Software as a craft"
+video-url: //www.slideshare.net/slideshow/embed_code/key/ltUJHbkVpru37U
+date: 2018-02-23 10:00:00 +00:00
+---
+
+## Mute talk: Software as a craft
+
+This talk happened at the University of  Tarragona (URV) with a lot of students as attendees. It encourages to strive for a learning culture and being better professionals in software development.
```

Comment:
These are the slides of the presentation. I've also included some extra slides that explain what the attendees answered, so you can get a better, fuller experience reading them.

Reply:
Thanks @alvarogarcia7 . I'm repeating this PR with _silent_, instead of _mute_. I'm going to follow your recommendations. Thanks!